transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransportImpl.kt

```diff
@@ -1,32 +1,19 @@
 package com.genesys.cloud.messenger.transport.core
 
-import com.genesys.cloud.messenger.transport.network.DEFAULT_PING_INTERVAL_IN_SECONDS
 import com.genesys.cloud.messenger.transport.network.DeploymentConfigUseCase
 import com.genesys.cloud.messenger.transport.network.PlatformSocket
 import com.genesys.cloud.messenger.transport.network.ReconnectionHandlerImpl
 import com.genesys.cloud.messenger.transport.network.WebMessagingApi
 import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
-import com.genesys.cloud.messenger.transport.util.DefaultTokenStore
 import com.genesys.cloud.messenger.transport.util.logs.Log
 import com.genesys.cloud.messenger.transport.util.logs.LogTag
 
-/**
- * Client entry point for the Mobile Messenger Transport SDK.
- */
-object MobileMessenger {
+internal class MessengerTransportImpl(private val configuration: Configuration) {
 
-    /**
-     * Creates an instance of [MessagingClient] based on the provided configuration.
-     *
-     * @param configuration the configuration parameters for setting up the client.
-     */
-    fun createMessagingClient(
-        configuration: Configuration,
-    ): MessagingClient {
+    fun createMessagingClient(token: String): MessagingClient {
         val log = Log(configuration.logging, LogTag.MESSAGING_CLIENT)
         val api = WebMessagingApi(configuration)
-        val webSocket = PlatformSocket(log.withTag(LogTag.WEBSOCKET), configuration.webSocketUrl, DEFAULT_PING_INTERVAL_IN_SECONDS)
-        val token = DefaultTokenStore(configuration.tokenStoreKey).token
+        val webSocket = PlatformSocket(log.withTag(LogTag.WEBSOCKET), configuration.webSocketUrl, 300000)
```

Comment:
I think we should use the DEFAULT_PING_INTERVAL_IN_SECONDS here

Reply:
Yeah, reimplementation issue.